tests/testthat/test-spacetime-constructor_cli_update_patch.R

```diff
@@ -0,0 +1,13 @@
+test_that("local_g_perm", {
+  df_fp <- system.file("extdata", "bos-ecometric.csv", package = "sfdep")
+  geo_fp <- system.file("extdata", "bos-ecometric.geojson", package = "sfdep")
+
+  # read in data
+  df <- read.csv(
+    df_fp, colClasses = c("character", "character", "integer", "double", "Date") # makes .region_id chr to match geo's chr .region_id
```

Comment:
Can you make the `df` assignment like so:

```r
# read in data
df <- read.csv(
  df_fp, colClasses = c("numeric", "character", "integer", "double", "Date")
)
```

so that we get an error and then change the expectation to `expect_error()` so that we can make sure the eror is thrown appropriately?